packages/front-end/components/Tags/ProjectTags.tsx

```diff
@@ -0,0 +1,57 @@
+import { useDefinitions } from "../../services/DefinitionsContext";
+import Tag from "./Tag";
+
+export interface Props {
+  projectIds?: string[];
+  sort?: boolean;
+  className?: string;
+}
+
+export default function ProjectTags({
+  projectIds,
+  sort = true,
+  className = "badge-ellipsis short",
+}: Props) {
+  const { projects, project } = useDefinitions();
+  if (!projectIds) {
+    return (
+      <Tag
```

Comment:
We shouldn't use the `<Tag>` component for this.  We can make a new `<Badge>` UI component that both projects and tags can share if we want to share some CSS classes between them.

Reply:
added a <Badge> component, made the other related components work with this.